src/test/shared/vscode/message.ts

```diff
@@ -130,7 +130,7 @@ export class TestMessage<T extends vscode.MessageItem = vscode.MessageItem> {
             const mappedItems = items.map(i => (typeof i === 'string' ? { title: i } : i))
             const testMessage = new TestMessage(message, {
                 severity,
-                modal: opt.modal,
+                modal: opt?.modal,
```

Comment:
I think we should turn on strict indexing at some point. It makes elements possibly `undefined` unless you check the length.